stripper/ze_cyberspace_v1.cfg

```diff
@@ -1,3 +1,30 @@
+;prevent the map from changing freezetime
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	delete:
+	{
+		"OnMapSpawn" "ScommandCommandmp_freezetime 00-1"
+	}
+}
+
+;lazy way to patch stage 3 nuke avoidance spot(s)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "c3_end_lv_change"
+	}
+	insert:
+	{
+		"OnTrigger" "playerRunScriptCodelocal pos=self.GetOrigin();foreach(k,_ in{SetHealth=0}){if(pos.x>-9184)EntFireByHandle(self,k,(0).tostring(),0,null,null)}10-1"
```

Comment:
trigger_once outputs should be only once, and OnStartTouch is generally preferred.